.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "libs/orleans"]
```

Comment:
TODO: delete current `libs/*.dll`, I just didn't want to create noise in this PR with 300+ deletes.,